Refactor(eos_cli_config_gen, eos_designs)!: Moved the NTP server vrf key outside to servers list (#5905)
aristanetworks/ansible-avd: docs/porting-guides/6.x.x.md

+     - name: tapagg
```

> ### `ntp.servers[].vrf` moved to `ntp.vrf` since all NTP servers must use the same VRF

Reviewer: once you add the removed in the schema please also add this to the table

Author: done

In AVD 5.x, the VRF for NTP servers was defined individually under `ntp.servers[].vrf`.

> Starting with AVD 6.0.0, all NTP servers must reside in the same VRF. Therefore, `ntp.servers[].vrf` has been replaced by `ntp.vrf`.

Reviewer: ```suggestion
Starting with AVD 6.0.0, because in EOS all NTP servers must be in the same VRF, `ntp.servers[].vrf` has been replaced by `ntp.vrf` to avoid misconfigurations.
```

+     - name: tapagg
```

### `ntp.servers[].vrf` moved to `ntp.vrf` since all NTP servers must use the same VRF

In AVD 5.x, the VRF for NTP servers was defined individually under `ntp.servers[].vrf`.

Starting with AVD 6.0.0, because in EOS all NTP servers must be in the same VRF, `ntp.servers[].vrf` has been replaced by `ntp.vrf` to avoid misconfigurations.

```diff

> ntp:
>   servers:
>     - name: 2.2.2.55
> -     vrf: VRF1
> + vrf: VRF1

Reviewer: ```suggestion
ntp:
+ vrf: VRF1
  servers:
    - name: 2.2.2.55
-     vrf: VRF1
```
